std/math/hardware.d

```diff
@@ -159,6 +162,15 @@ private:
             return result;
             `);
         }
+        else version (LoongArch_Any)
+        {
+            uint result = void;
+            asm pure nothrow @nogc
+            {
+                "movfcsr2gr %0,$r2" : "=r" (result);
+            }
+            return result & EXCEPTIONS_MASK;
+        }
```

Comment:
Is there a reason why this differs from the patch posted to gcc-patches?  What gets downstreamed to gdc would preferably be an identical copy.